src/main/scala/nodes/learning/KernelBlockModel.scala

```diff
@@ -0,0 +1,74 @@
+package nodes.learning
+
+import breeze.linalg._
+import edu.berkeley.cs.amplab.mlmatrix.{RowPartition, NormalEquations, BlockCoordinateDescent, RowPartitionedMatrix}
+import nodes.stats.{StandardScalerModel, StandardScaler}
+import nodes.util.{VectorSplitter, Identity}
+import org.apache.spark.broadcast.Broadcast
+import org.apache.spark.rdd.RDD
+import utils.{MatrixUtils, Stats}
+import workflow.{Transformer, LabelEstimator}
+
+
+/**
+ * @param models different kernel models for different lambdas
+ * @param blockSize blockSize to split data before applying transformations
+ * @param lambdas  lambdas corresponding to the models
+ * @param in the training Data
+ * @param kernelGen the kernel generator
+ * @param nTrain number j training examples
+ **/
+
+class KernelBlockModel(
+  model: DenseMatrix[Double],
```

Comment:
So the model won't fit in a single matrix if the data is large due to Java array size limits. For example if you have 2M examples and 1000 classes, the size of the model is `n*k` which is 2 billion elements and very close to the java array limit of 2^31 - 1.  We should pass in a blocked model (similar to BlockLinearMapper)